getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -241,12 +460,14 @@ characters in a given string. See the examples below:
     # Both quote styles would require 2 escapes; prefer double quotes if it's a tie.
     print("'hello' \"world\"")
 
+.. _naming_conventions:
+
```

Comment:
Is this addition supposed to be here?

Reply:
Yes it's how you link to sections in rst